nvflare/lighter/cc/image_builder/config/cc_params.yml

```diff
@@ -0,0 +1,19 @@
+computer_env: onprem_cvm
+cc_cpu_mechanism: amd_sev_snp
```

Comment:
I know sev that device will be ready from the base image, what about the snpguest binary, should that also coming from the base image?